functional/security/ssl-tests/playlist.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TKG/playlist.xsd">
+	<test>
+		<testCaseName>ssl-tests</testCaseName>
+		<command>$(JAVA_COMMAND) -Xmx512m -jar $(Q)$(TEST_RESROOT)$(D)jtreg$(D)lib$(D)jtreg.jar$(Q) \
+		-a -xml -v:fail,error,time,nopass,summary -timeoutFactor:2 \
+		-w $(Q)$(REPORTDIR)$(D)work$(Q) \
+		-r $(Q)$(REPORTDIR)$(D)report$(Q) \
+		-jdk:$(Q)$(TEST_JDK_HOME)$(Q) \
+		$(Q)$(TEST_ROOT)$(D)functional$(D)security$(D)ssl-tests$(D)ssl-tests$(D)jtreg-wrappers$(Q); \
+		$(TEST_STATUS)</command>
+		<levels>
+			<level>extended</level>
```

Comment:
Since they are new tests, we may want to consider bringing them into `<level>dev</level>`.  
Will discuss this with other reviewers for their thoughts.

Reply:
I updated level to dev.